src/locale/messages.fr.xlf

```diff
@@ -355,7 +355,7 @@
       </trans-unit>
       <trans-unit id="2569213046451523868" datatype="html">
         <source>No previous tests found for this domain.</source>
-        <target state="translated">Aucun historiqutest n'a été trouvé pour ce domaine.</target>
+        <target state="translated">Aucun test n'a été trouvé pour ce domaine.</target>
```

Comment:
This sentence is better translated as “Aucun test antérieur n’a été trouvé pour ce domaine”.

Reply:
Done